packages/server/src/resources/resources.controller.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  CACHE_MANAGER,
+  Controller,
+  Get,
+  HttpService,
+  Inject,
+  Param,
+} from '@nestjs/common';
+import { Cache } from 'cache-manager';
+import { CourseModel } from '../course/course.entity';
+import { Connection } from 'typeorm';
+
```

Comment:
are you sure we wouldn't want a JWT token at least? People can only access this info if they are in an authorized session to use the app?

Reply:
if it's the fullcal lib making the request, would it have a JWT token? when i was writing this it seemed like putting the endpoint in any other controller caused guard issues